packages/web/src/scss/components/Header/_Header.scss

```
@@ -1,13 +1,12 @@
-@use '../../tools/accessibility';
+// 1. Use a `::after` pseudo element to create a decorative border.
```

```suggestion
// 1. Use an `::after` pseudo-element to create a decorative border.
```
Imho better like this